modules/gitops-release-notes-1-9-2.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-9-2_{context}"]
+= Release notes for {gitops-title} 1.9.2
+
+{gitops-title} 1.9.2 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-9-2_{context}"]
+== Errata updates
+
+=== RHSA-2023:120042 - {gitops-title} 1.9.2 security update advisory
+
+Issued: 2023-09-07
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://errata.engineering.redhat.com/advisory/120042[RHBA-2023:120042]
```

Comment:
```suggestion
* link:https://access.redhat.com/errata/120042[RHBA-2023:120042]
```

Reply:
@masauso-lungu Please remember to notice the change in the RHSA number if any for this advisory once the errata state is changed to SHIPPED LIVE.